### PR TITLE
Correct load_external documentation to match function call

### PR DESCRIPTION
**Pull request by iolivergithub**
Note, also changed the parameters order to allow in_sensitive to be None.
Changed in_private to be in_sensitive to match other functions names

### CODE REVIEW

**Comment by whooo** [comment]
I'm pretty sure that this change will break some tests, have you run the unit tests locally?

**Comment by iolivergithub** [comment]
I reverted the parameter order change.    953abb9d7d98ddd5ac5e6320db0bd648f83b70d2

**Comment by iolivergithub** [comment]
----------- coverage: platform linux, python 3.9.5-final-0 -----------
Name                               Stmts   Miss Branch BrPart  Cover
--------------------------------------------------------------------
tpm2_pytss/ESAPI.py                 1216     14    136     11    98%
tpm2_pytss/FAPI.py                   472     91     72     15    78%
tpm2_pytss/TCTI.py                    52     14     10      1    66%
tpm2_pytss/TCTILdr.py                 48      5     14      1    90%
tpm2_pytss/TSS2_Exception.py          42      0      8      1    98%
tpm2_pytss/__init__.py                11      2      0      0    82%
tpm2_pytss/callbacks.py               15      0      6      2    90%
tpm2_pytss/constants.py              992      3    124      4    99%
tpm2_pytss/fapi_info.py               74     41     22      1    35%
tpm2_pytss/internal/constants.py       8      0      0      0   100%
tpm2_pytss/internal/crypto.py        311      3     96      3    99%
tpm2_pytss/internal/utils.py         110      4     64      7    94%
tpm2_pytss/tsskey.py                 121      1     24      1    99%
tpm2_pytss/types.py                  863     19    256     20    97%
tpm2_pytss/utils.py                   52      0     10      3    95%
--------------------------------------------------------------------
TOTAL                               4387    197    842     70    94%


============================================================ 393 passed, 20 skipped in 193.91s (0:03:13) ============================================================

File: tpm2_pytss/ESAPI.py [hunk 1]
```diff
@@ -823,7 +823,7 @@ def load(
 
     def load_external(
         self,
-        in_private: TPM2B_SENSITIVE,
+        in_sensitive: TPM2B_SENSITIVE,
         in_public: TPM2B_PUBLIC,
```
Comment (whooo) [inline tpm2_pytss/ESAPI.py:827]: the argument to Esys_LoadExternal / TPM2_LoadExternal is called inPrivate, so it should be left as in_private

File: tpm2_pytss/ESAPI.py [hunk 2]
```diff
@@ -837,8 +837,8 @@ def load_external(
         available
 
         Args:
-            in_private (TPM2B_PRIVATE): The private portion of the object.
             in_public (TPM2B_PUBLIC): The public portion of the object.
+            in_sensitive (TPM2B_SENSITIVE): The sensitive object. Use None if no sensitive object is required (this is the default)            
             hierarchy (ESYS_TR): Hierarchy with which the object area is associated.
```
Comment (whooo) [inline tpm2_pytss/ESAPI.py:842]: same here

File: tpm2_pytss/ESAPI.py [hunk 3]
```diff
@@ -864,7 +864,7 @@ def load_external(
         _check_handle_type(session3, "session3")
 
         in_private_cdata = _get_cdata(
-            in_private, TPM2B_SENSITIVE, "in_private", allow_none=True
+            in_sensitive, TPM2B_SENSITIVE, "in_sensitive", allow_none=True
         )
```
Comment (whooo) [inline tpm2_pytss/ESAPI.py:868]: same here

**Comment by whooo** [comment]
You should squash the commits as well

**Comment by williamcroberts** [comment]
> I reverted the parameter order change. [953abb9](https://github.com/tpm2-software/tpm2-pytss/commit/953abb9d7d98ddd5ac5e6320db0bd648f83b70d2)

I was actually interested in that part of the change, the re-order with in_private set to None as a default makes loading public keys dirt simple. However, now the change is just the re-name, and as pointed out, in_private keeps it consistent with the lower API.

**Comment by williamcroberts** [comment]
Closing in favor of commits on:
https://github.com/tpm2-software/tpm2-pytss/pull/275

**Pull request closed without merging**